tests/kubeaddons-enterprise/kuttl-test.yaml

```diff
@@ -2,20 +2,26 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestSuite
 commands:
-  - command: docker cp kind-control-plane:/etc/kubernetes/pki/ca.crt /tmp/ca.crt
-  - command: docker cp kind-control-plane:/etc/kubernetes/pki/ca.key /tmp/ca.key
+  # Install KUDO out-of-band from Kubeaddons to remove cert-manager dependency.
+  # We're installing kubeaddons version mesosphere/kubeaddons:latest, below. 
+  # Kubeaddons pins the version of KUDO used here: https://github.com/mesosphere/kubeaddons/blob/master/go.mod#L25
+  # KUDO version specified below must be API compatible below.
   - command: >
-      bash -c "kubectl create namespace cert-manager --dry-run=client -o yaml |
-      kubectl apply -f -"
+      bash -c "
+        # Avoid expensive redownload of the KUDO CLI.
+        if [[ ! -x kubectl-kudo ]]; then
+          # Update KUDO version here as needed.
+          wget -O kubectl-kudo http://github.com/kudobuilder/kudo/releases/download/v0.18.0/kubectl-kudo_0.18.0_linux_x86_64
```

Comment:
We can parameterize kudo version as an env var and place it in `run-tests.sh`. This will help kudo version bumps to come in :eye: easily. Overall, LGTM!

Reply:
@rishabh96b Oh, let's add that to the next PR if this stablizes the Dispatch builds.